### PR TITLE
fix: missing commands help information

### LINKED ISSUE #738: Some commands missing in the help output
While working with the `clarity-repl` I figured out that some commands are missing in the help information.

`get_epoch`
`encode`
`decode`

Is that a known issue or is it mean to be like that?

image

Cheers!

### PR DESCRIPTION
**Pull request by csgui**
Fix #738 .

And, when reviewing, also please check if the command wording/spelling is ok.

Cheers!

### CODE REVIEW

File: components/clarity-repl/src/repl/session.rs [hunk 1]
```diff
@@ -836,6 +840,14 @@ impl Session {
             "{}",
             help_colour.paint("::read <filename>\t\t\tRead expressions from a file")
         ));
+        output.push(format!(
+            "{}",
+            help_colour.paint("::encode <expr>\t\t\t\tEncode an expression to a Clarity Value bytes representation")
+        ));
+        output.push(format!(
+            "{}",
+            help_colour.paint("::decode <expr>\t\t\t\tDecode a Clarity Value")
```
Comment (hugocaillard) [inline components/clarity-repl/src/repl/session.rs:849]:
```suggestion
            help_colour.paint("::decode <bytes>\t\t\t\tDecode a Clarity Value bytes representation")
```

Could we make it easier to understand what is expected here?

**Review by hugocaillard — COMMENTED** [review]
Thanks for the addition!
Small suggestion, let me know what you think

File: components/clarity-repl/src/repl/session.rs (hunk 1 above)
Comment (csgui) [reply components/clarity-repl/src/repl/session.rs]: Sure... better description indeed. Fixed.

**Review by hugocaillard — APPROVED** [review]
LGTM!
Thanks a lot for the addition!

**Comment by csgui** [comment]
Failing check will be fixed by PR #757 .

**Pull request merged**